src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedCOMReference.xaml

```diff
@@ -66,15 +66,6 @@
         <EnumValue Name="Native Assembly" DisplayName="Native Assembly" />
     </EnumProperty>
 
-    <StringProperty Name="Identity"
```

Comment:
I don't think @lifengl was referring to any of the rules that represent "target results". These aren't used in the "file context" as called out in the bug.

Reply:
You can see the context here: https://github.com/dotnet/project-system/blob/ee188a4b5c4d03c63f10f461810e7352e6cf85db/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets#L164.

Reply:
Yeah, I wasn't sure about these - just figured that I might as well do all of the ones that were easy to do.  These could be the source dependencies node issues though.